client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -19,20 +21,6 @@ const useSteps = ( { flowName, hasPaidDomain, isDestinationSetupSiteFlow } ) =>
 		case 'launch-site':
 			steps = [ __( 'Your site will be live shortly.' ) ]; // copy from 'packages/launch/src/focused-launch/success'
 			break;
-		case 'setup-site':
```

Comment:
Why not just add a case here and mention that this case is for design picker related flows if required?
``` javascript
/* Flows with design picker */
case 'setup-site':
case 'do-it-for-me':
```